app/services/organization_update_service.rb

```diff
@@ -11,14 +11,16 @@ class << self
     # @return [Boolean]
     def update(organization, params)
       return false unless valid?(organization, params)
+      # return organization unless valid?(organization, params)
```

Comment:
Please don't check in commented-out code :smile:

Reply:
Oops!  I missed that one, good catch!